src/perftest_resources.c

```diff
@@ -1579,6 +1556,9 @@ int ctx_init(struct pingpong_context *ctx, struct perftest_parameters *user_para
 	for (i=0; i < user_param->num_of_qps; i++) {
 		if (create_qp_main(ctx, user_param, i, num_of_qps)) {
 			fprintf(stderr, "Failed to create QP.\n");
+			if (user_param->connection_type == DC) {
+				fprintf(stderr, "DC is not supported for the divice chosen.\n");
```

Comment:
Is this related to the patch? Also there's a typo in device.

Reply:
> Is this related to the patch? Also there's a typo in device.

Yes, it is, because the reason of failure of DC QP creation is fact that only new API supports DC.